graphql.go

```diff
@@ -233,6 +233,10 @@ func (s *Schema) ValidateWithVariables(queryString string, variables map[string]
 		return []*errors.QueryError{qErr}
 	}
 
+	if len(doc.Operations) == 0 {
+		return []*errors.QueryError{errors.Errorf("executable document must contain at least one operation")}
+	}
```

Comment:
This helps to exit early when testing with an executable document without any operations.